src/Components/Banner.js

```diff
@@ -1,5 +1,36 @@
 import React from "react";
+import styles from "../Css/Banner.module.css";
 
-export default function Banner() {
-  return <></>;
+export default function App() {
+  return (
+    <div className={`${styles.App}`}>
+      <div className={`${styles.main}`}>
+        <div className={`${styles.head}`}>
+          Give wings to
+          <br />
+          your <span className={`${styles.dreams}`}>dreams</span>
```

Comment:
Use className={style.dreams} instead of {`${styles.dreams}}.
Change it for remaining classNames also which have only custom CSS.